3.resistance-signature/scripts/nbconverted/0.training-test-split.py

```diff
@@ -267,54 +267,61 @@
 # In[12]:
 
 
+### Remove Batch 3 ###
 # Load inference data (a different hold out)
-inference_batch = "2019_06_25_Batch3"
-inference_file = pathlib.Path(f"../3.bulk-signatures/data/{inference_batch}_combined_normalized.csv.gz")
-inference_df = pd.read_csv(inference_file)
+#inference_batch = "2019_06_25_Batch3"
+#inference_file = pathlib.Path(f"../3.bulk-signatures/data/{inference_batch}_combined_normalized.csv.gz")
+#inference_df = pd.read_csv(inference_file)
 
-inference_df = inference_df.assign(
-    Metadata_dataset="untreated_mystery_clones",
-    Metadata_batch=inference_batch,
-    Metadata_clone_type="resistant",
-    Metadata_clone_type_indicator=1,
-    Metadata_model_split="inference"
-)
+#inference_df = inference_df.assign(
+    #Metadata_dataset="untreated_mystery_clones",
+    #Metadata_batch=inference_batch,
+    #Metadata_clone_type="resistant",
+    #Metadata_clone_type_indicator=1,
+    #Metadata_model_split="inference"
+#)
 
-inference_df.loc[inference_df.Metadata_clone_number.str.contains("WT"), "Metadata_clone_type"] = "sensitive"
-inference_df.loc[inference_df.Metadata_clone_number.str.contains("WT"), "Metadata_clone_type_indicator"] = 0
-inference_df = inference_df.assign(
-    Metadata_unique_sample_name=[f"profile_{x}_inference" for x in range(0, inference_df.shape[0])]
-)
+#inference_df.loc[inference_df.Metadata_clone_number.str.contains("WT"), "Metadata_clone_type"] = "sensitive"
+#inference_df.loc[inference_df.Metadata_clone_number.str.contains("WT"), "Metadata_clone_type_indicator"] = 0
+#inference_df = inference_df.assign(
+#    Metadata_unique_sample_name=[f"profile_{x}_inference" for x in range(0, inference_df.shape[0])]
+#)
 
-inference_df.Metadata_clone_number.value_counts()
+#inference_df.Metadata_clone_number.value_counts()
 
 
 # In[13]:
 
 
 # Combine profiles into a single dataset and output
-bortezomib_df = pd.concat(
-    [
-        full_df.query("Metadata_dataset == 'bortezomib'"),
-        inference_df
-    ],
-    axis="rows",
-    sort=False
-).reset_index(drop=True)
+#bortezomib_df = pd.concat(
+ #   [
+  #      full_df.query("Metadata_dataset == 'bortezomib'"),
+   #     inference_df
+    #],
+    #axis="rows",
+    #sort=False
+#).reset_index(drop=True)
 
-output_file = pathlib.Path(f"{output_dir}/bortezomib_signature_analytical_set.tsv.gz")
-bortezomib_df.to_csv(output_file, sep="\t", index=False)
+#output_file = pathlib.Path(f"{output_dir}/bortezomib_signature_analytical_set.tsv.gz")
+#bortezomib_df.to_csv(output_file, sep="\t", index=False)
 
 
 # In[14]:
 
 
-print(bortezomib_df.shape)
-bortezomib_df.head()
+#print(bortezomib_df.shape)
+#bortezomib_df.head()
 
 
 # In[15]:
 
 
-assert len(bortezomib_df.Metadata_unique_sample_name.unique()) == bortezomib_df.shape[0]
+#assert len(bortezomib_df.Metadata_unique_sample_name.unique()) == bortezomib_df.shape[0]
+
+
+# In[ ]:
```

Comment:
This is how the extra new lines show up on an nbconverted file. Please delete in the corresponding `.ipynb` - the nbconverted files should never be edited!